### ISSUE TITLE
Deserialization error on Pricing plans with null Price per Unit or Flat Fee

### ISSUE THREAD

**Issue by CoastalP47** [stripe/stripe-dotnet#1375]
Building out a test "create customer with subscription and products" method I kept getting a deserialization error when I would attempt to create a subscription with plans that had either nulls in the _price per unit_ or _flat fee_. The UI allows for creating those kinds of plans and doesn't throw an error in that instance. However, if I change those values from null to 0 I can go through the process successfully.

It would appear that those fields need to be nullable.

![image](https://user-images.githubusercontent.com/5049511/48136286-4878f780-e26d-11e8-8023-0503b477895b.png)

Sample method:
```c#
// [...] abbreviated for brevity
// create subscription
var subscriptionItems = new List<SubscriptionItemOption>();
foreach (var product in configuration.DefaultSubscriptionProducts)
    subscriptionItems.Add(new SubscriptionItemOption {
        PlanId = product,
        Quantity = 0
    });
var subscription = subscriptionService.Create(new SubscriptionCreateOptions {
    CustomerId = customer.Id,
    Items = subscriptionItems,
    BillingCycleAnchor = beginningOfNextMonth.DateTime,
    Metadata = metadata,
});
// [...]
```

**OS Version**: Windows 10
**.net Version**: .net core 2.0
**Stripe Version**: 20.4.0

**Comment by ob-stripe** [comment]
Hey @CoastalP47, thanks for the report. You are correct, this field should be nullable. We'll release a fix soon. Sorry for the inconvenience!

**Comment by ob-stripe** [comment]
Fixed in 21.0.0.
